xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h

```diff
@@ -192,17 +192,22 @@ namespace gui
     ///
     /// @return The current icon image path (if present)
     ///
+    /// @warning Usage of GetIconImage is deprecated and will soon be removed
+    /// use \ref GetArt instead
+    ///
     std::string GetIconImage()
```

Comment:
This PR breaks `kodi.binary.global.gui` compatibility, which means at least a rebuild is required.
Because of that you could just remove this function.